### ISSUE TITLE
Upgrade from 4.8.0 - 4.9.0: unterminated string literal error during svelte-check, build fails

### ISSUE THREAD

**Issue by mlandisbqs** [sveltejs/svelte-preprocess#402]
**Describe the bug**
Isolated to 4.9.0 code changes (4.8.0 compiles the same code without the bug):
https://github.com/sveltejs/svelte-preprocess/compare/v4.8.0...v4.9.0?short_path=61c21cb#diff-96443e2ca98f566a13837889beb8b5a9da9d97c396ad64bc3a28dbf0e3557f37

After the upgrade -- the generated code contains unterminated string literal which typescript then fails to compile. 
This happens right here in the code (line 140, src/transformers/typescript.ts):

![image](https://user-images.githubusercontent.com/39845713/132403915-c161caf0-24bb-4f45-a07e-1211b637d119.png)

^-- the codestores array includes an entry like this ``["'"]``

**Logs**
```
rc/components/script-response/input-type/single-input-type.svelte.injected.ts:74:224 - error TS1002: Unterminated string literal.

74 const $$vars$$ = [sessionStore,sessionStore,',promptResponse,prompt,$sessionStore,sessionStore,prompt,promptResponse,BqpTextField,label,type,error,optional,unit,textarea,inputAttrs,prompt,value,persistChange,persistChange];
                                                                                                                                                                                                                                  

src/components/script-response/input-type/range-input-type.svelte.injected.ts:89:209 - error TS1002: Unterminated string literal.

89 const $$vars$$ = [',promptResponses,BqpGroup,highlight,$sessionStore,sessionStore,prompt,BqpTextField,type,unit,fromValue,persistChange,BqpTextField,type,unit,fromValue,toValue,isValid,toValue,persistChange];
                                                                                                                                                                                                                   
```

**Expected behavior**
codestores should be captured without a single quote value in the array.

**Information about your project:**

- Your browser and the version: (e.x. Chrome 52.1, Firefox 48.0, IE 10)

- Your operating system: (e.x. OS X 10, Ubuntu Linux 19.10, Windows XP, etc)
MacOs 10.15.4

- `svelte-preprocess` version (Please check you can reproduce the issue with the latest release!)
4.9.0 (same problem on latest release)

- Whether your project uses Webpack or Rollup
rollup

**Comment by mlandisbqs** [comment]
This looks like it could be related to #401

**Comment by kaisermann** [comment]
Hey, @mlandisbqs 👋 Thanks for the report! Can you share your file content or at least all identifiers beginning with `$`?

**Comment by mlandisbqs** [comment]
ok - I actually think I see the issue... both of the files that have problems have a string literal with a dollar sign...

![image](https://user-images.githubusercontent.com/39845713/132514472-eef237f8-738c-4b26-8a07-0b009159c987.png)

...I haven't tested removing it yet, but that must be the issue. I'm not sure that I can share the whole component file.

**Comment by kaisermann** [comment]
That's more than enough and is probably the culprit. This helps a lot, thanks!

**Comment by seanlail** [comment]
@kaisermann I'm sure you have enough info, but just in case:

I just ran into the same thing with this code:
```
<script lang="ts">
  export let code = "gbp";

  const symbols: Record<string, string> = { eur: "€", gbp: "£", usd: "$" };

  $: symbol =
    !!code && symbols[code.toLowerCase()] ? symbols[code.toLowerCase()] : "";
</script>

{symbol}
```


```
src/components/CurrencySymbol.svelte.injected.ts:11:29 - error TS1002: Unterminated string literal.
11 const $$vars$$ = [",symbol];
```

**Comment by seanlail** [comment]
This breaks in 4.8.0 for me. Only 4.7.4 and older worked.

**Comment by dummdidumm** [comment]
Found the culprit, will prepare a PR shortly

**Comment by kaisermann** [comment]
Fix released in v4.9.4 🎉 . Thanks @dummdidumm!
